ide/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+	id 'com.diffplug.gradle.oomph.ide' version '3.1.0'
+}
+
+oomphIde {
+	splash 'typefox.png'
+
+	// JDT neon
+	repoEclipse '4.6.0'
+	jdt {}
+
+	// add syntax highlighting for gradle scripts
+	repo    'http://www.nodeclipse.org/updates/gradle-ide-pack/'
+	feature 'org.nodeclipse.enide.editors.gradle.feature'
```

Comment:
This one should be replaced with the latest milestone of buildship 2.0 (http://download.eclipse.org/buildship/updates/e45/milestones/2.x/).